index.d.ts

```diff
@@ -0,0 +1,47 @@
+import { MutableRefObject, ReactElement, Ref } from "react";
+
+import * as CSS from "csstype";
+
+export as namespace Showtime;
+
+export type PredefinedTransition = "slideFade" | "slide" | "fade" | "rise" | "scale"
+
+export interface TransitionProperties {
```

Comment:
Strictly speaking, this might be better named as singular `TransitionProperty` since it represents a single CSS property that will be transitioned. That also matched the "CSS Property" term used to describe it in the API section of the Readme.